tests/utils/shopper.js

```diff
@@ -293,5 +300,50 @@ export const shopper = {
 				}
 			);
 		},
+
+		setCartQuantity: async ( productTitle, quantityValue ) => {
+			const cartItemXPath = getCartItemExpression( productTitle );
+			const quantityInputXPath =
+				cartItemXPath + '//' + getQtyInputExpression();
```

Comment:
I noticed that `getCartItemExpression` returns the x path with `//` at the start, but `getQtyInputExpression` doesn't include the `//`. This was a bit misleading to me, I would suggest consistency with the expression functions here for clarity

Reply:
`getQtyInputExpression ` is also reused in the `getQtyItemExpression` function in `tests/utils/expressions.js`:

```js
export const getQtyItemExpression = ( args ) =>
	'div[@class="wc-block-cart-item__quantity" and div[@class="wc-block-components-quantity-selector" and ' +
	getQtyInputExpression( args ) + // =====> Here
	']]';
```

In the code snippet above, if `getQtyInputExpression` starts with the axis `//` this will return unwanted result.

Reply:
Ok cool, let's leave it as is then